edgedb/codegen/async_query.py.template

```diff
@@ -0,0 +1,12 @@
+from __future__ import annotations
+{gen.imports}
+
+
+{gen.aliases}{gen.definitions}async def {stem}(
+    client: edgedb.AsyncIOClient,
+{in_type_args}){out_type}:
+    return await client.query(
+        """\
+        {query}\
+        """,
+{in_type_call}    )
```

Comment:
The formatting here looks a bit weird, is that on purpose? Can you show an example of generated code?

Reply:
```python
from __future__ import annotations
import dataclasses
import edgedb
import typing
import uuid


TryEnum0 = str


@dataclasses.dataclass
class TryEnum1:
    id: uuid.UUID

    @classmethod
    def __get_validators__(cls):
        return []


def try_enum(
    client: edgedb.Client,
    a: TryEnum0,
    b: edgedb.Object,
) -> typing.Tuple[TryEnum0, TryEnum1]:
    return client.query(
        """\
        select (<MyString>$a, <User>$b)\
        """,
        a=a,
        b=b,
    )
```

Schema:

```
    scalar type MyString extending str;

    type User {...}
```

Reply:
OK, looks nice!